docs/standard/serialization/binary-serialization.md

```diff
@@ -126,7 +126,7 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.UInt64?displayProperty=fullName>   
 - <xref:System.UIntPtr?displayProperty=fullName>   
 - <xref:System.Uri?displayProperty=fullName>   
-- <xref:System.ValueTuple?displayProperty=fullName>   
+- <xref:System.ValueTuple?displayProperty=fullName> (not supported in .NET Framework < 4.7.1)  
```

Comment:
@ViktorHofer, could you change this to "(not serializable in .NET Framework 4.7 and earlier versions)"?

Reply:
absolutely.